packages/react/src/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,192 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { ChevronLeftIcon, ChevronRightIcon } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes, useEffect, useState } from 'react'
+import { Icon } from '../Icon/Icon'
+
+export interface PaginationProps extends HTMLAttributes<HTMLElement> {
+  /**
+   * The number of items in your paginated collection.
+   *
+   * Note that this is not the same as the number of pages, but will be used to calculate it.
+   */
+  collectionSize: number
```

Comment:
why not offer a totalpages prop? This usually what is provided in a pagination object.

Reply:
I did a quick inventory, ASC, Ant, Carbon and Kolibri design systems all use something like `collectionSize` and `pageSize`. Atlassian passes an array with all pages(?) and Lion passes the total number of pages. Do you have other examples of Pagination components that only pass a `totalPages` prop?

Reply:
I checked MUI and Carbon React as well, and they basically ask for `count` or `totalItems`. It seems there are 2 variants of paginations. Like the navigation variant or the table variant. Were for the latter the props `collectionSize` and `pageSize` make more sense. 

- https://react.carbondesignsystem.com/?path=/docs/components-paginationnav--overview
- https://react.carbondesignsystem.com/?path=/docs/components-pagination--overview
- https://mui.com/material-ui/react-pagination/#basic-pagination
- https://mui.com/material-ui/react-pagination/#table-pagination